package.json

```diff
@@ -136,11 +136,12 @@
     "web components"
   ],
   "scripts": {
-    "build": "npm-run-all build:sass sass:render scssLint dist:js bundler postinstall build:api build:docs",
+    "build": "npm-run-all build:sass sass:render build:api test build:docs bundler build:demoScripts postinstall types",
```

Comment:
Shouldn't we be running the linters still? Without it here it never happens unless we do it manually.

Reply:
Adding that back in now